Formula/mitmproxy.rb

```diff
@@ -115,9 +110,14 @@ class Mitmproxy < Formula
     sha256 "7f91197cc9e48f989d12e4e6fbc46495c446636dfc81b9ccf50bb0ec74b91d4b"
   end
 
+  resource "mitmproxy-wireguard" do
+    url "https://files.pythonhosted.org/packages/8e/62/c438e16254c947732031d4132993731174e6471b199c62ed4f4c9f5bc7fb/mitmproxy_wireguard-0.1.14.tar.gz"
+    sha256 "faf260f98f95d3944d522c75354dd23e0a3792099c71facb2a5f69ad8c0aa369"
+  end
```

Comment:
```
    💥 maturin failed
      Caused by: Cargo metadata failed. Does your crate compile with `cargo build`?
      Caused by: `cargo metadata` exited with an error: error: failed to load manifest for workspace member `/private/tmp/mitmproxy--mitmproxy-wireguard-20221028-28659-1argyc1/mitmproxy_wireguard-0.1.14/test-client`
  
    Caused by:
      failed to read `/private/tmp/mitmproxy--mitmproxy-wireguard-20221028-28659-1argyc1/mitmproxy_wireguard-0.1.14/test-client/Cargo.toml`
```

Reply:
`mitmproxy_wireguard-0.1.15` should fix the error shown here. :)